### PR TITLE
[DEV APPROVED] Add hero image and hero description component

### PR DESCRIPTION
**Pull request by tomas-stefano**
[TP Card](https://moneyadviceservice.tpondemand.com/entity/9163-fincap-article-page-view-or-add)

## Solution

* Add hero image component
* Add hero description component
* Update the cassettes in order to get the nodes of hero image and hero

Since with the hero component adds more components to the project I **needed to refactor the existing code**, but while I was doing I found two bugs on the current implementation which I decided to fix on the same branch.

* Fix a bug on cta component when the page does not have a cta link.
* Fix a bug on feedback email. When the email has `<p>` tags the mail app was opening with the `<p>` tags in the from field, so strip tags solves the issue.

### CODE REVIEW

File: app/presenters/article_presenter.rb [hunk 1]
```diff
@@ -30,6 +30,12 @@ def cta_links_content
...
 
   def feedback_content
-    feedback_block.try(:content)
+    view.strip_tags(feedback_block.try(:content))
+  end
+
+  def render_component(component_name, content:)
+    return if content.blank?
+
+    view.render("components/#{component_name}", content: content)
```
Comment (margOnline) [inline app/presenters/article_presenter.rb]: 😍

File: app/views/articles/show.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="l-constrained">
-  <div class="hero">
+  <div class="hero" style="background-image: url(<%= article.hero_image_component %>);">
```
Comment (margOnline) [inline app/views/articles/show.html.erb]: can the article image be displayed with an image tag rather than mixing css in with html?

Comment (tomas-stefano) [reply app/views/articles/show.html.erb]: I really wanted the same. I forgot to add on the description. I talked with @johnplayer1982 about it which he had a strong reasoning for it. @johnplayer1982 can you clarify again here, please?

**Comment by margOnline** [comment]
Please would you include in the description why the style attribute is inline.

**Comment by johnplayer1982** [comment]
The image is a background image because of the way we require it to scale across devices.  An image tag would result in a image which is always confined within the container it's in, not ideal for mobile view where the image would simply shrink.

Using a background image means that we can simply adjust the background position and scale it to fit either the height or width of the container element (but not both).  This provides us with extra flexibility on which images we use, and fixes the issue of varying screen widths.

**Pull request merged**